airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -126,6 +126,8 @@ def __init__(
         container_overrides: dict | None = None,
         array_properties: dict | None = None,
         node_overrides: dict | None = None,
+        share_identifier: str | None = None,
+        scheduling_priority_override: int | None = None,
```

Comment:
Please add `:param ` entries to the doc string for these new params

Reply:
added the new params